app/controllers/api_controller.rb

```diff
@@ -122,11 +122,11 @@ def set_access_control_headers
   end
 
   def base_config
-    Api::Settings.base
+    ManageIQ::API::Settings.base
```

Comment:
> If this is agreeable, moving forward controllers will live under a similar namespace, e.g.:
>
ManageIQ::API::BaseController
ManageIQ::API::VmsController

It was planned to look like :point_up: